.blueprint/generate-sample/templates/samples/imperative-ngx-jwt-maven-mysql-redis.jdl

```diff
@@ -4,6 +4,7 @@ application {
     authenticationType jwt
     baseName jhipsterSampleApplication
     buildTool maven
+    creationTimestamp 1617901618891
```

Comment:
Why is this necessary? I'm guessing it's not a required field.

Reply:
makes the diff in CI compare step smaller.